pkg/restic/common.go

```diff
@@ -95,13 +95,17 @@ func getPodSnapshotAnnotations(obj metav1.Object) map[string]string {
 	return res
 }
 
+func isPVBMatchPod(pvb *velerov1api.PodVolumeBackup, pod metav1.Object) bool {
+	return pod.GetName() == pvb.Spec.Pod.Name && pod.GetNamespace() == pvb.Spec.Pod.Namespace
```

Comment:
This is probably preference and I'm not  going to mark it as a blocker, but could we use [`NamespaceAndName`](https://github.com/vmware-tanzu/velero/blob/f42c63af1b9af445e38f78a7256b1c48ef79c10e/pkg/util/kube/utils.go#L36) to do the comparisons?

Reply:
Oh, duh, no we can't cause the PVB has the pod's namespace and name nested a layer deeper than that function is looking for.